src/main/router/router.test.ts

```diff
@@ -32,6 +33,7 @@ describe("router", () => {
     const di = getDiForUnitTesting({ doGeneralOverrides: true });
 
     mockFs();
+    di.permitSideEffects(fsInjectable);
```

Comment:
Why is this needed?

Reply:
Because the router tests still use `mockFs`

Reply:
Answer's good, resolved.